ajax/libs/react-widgets/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "react-widgets",
+  "description": "An à la carte set of polished, extensible, and accessible inputs built for React",
+  "filename": "react-widgets.min.js",
+  "author": {
+    "name": "Jason",
+    "email": "monastic.panic@gmail.com"
+  },
+  "keywords": [
+    "react",
+    "widgets",
+    "dropdown",
+    "combobox",
+    "calendar",
+    "datepicker",
+    "date picker",
+    "numberpicker",
+    "number picker",
+    "radio group",
+    "checkbox list",
+    "multiselect",
+    "form",
+    "input",
+    "react-ui",
+    "react-component"
+  ],
+  "homepage": "http://jquense.github.io/react-widgets/docs/",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jquense/react-widgets"
+  },
+  "npmName": "react-widgets",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "autoupdate": {
```

Comment:
As [CI](https://ci.cdnjs.com/cdnjs/cdnjs/10064) said this show multiple auto-update configs. We should remove one of it.
Because there are more release versions from npm(105) than GitHub(101), so we choice npm auto-update.
Let's remove the part as the followings:
```
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/jquense/react-widgets.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
    ]
  }
```
Thank you.